common/mariadb/templates/configmap-mariadb-maintenance.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   namespace: {{ $.Release.Namespace }}
-  name: mariadb-cronjob-maintenance
+  name: mariadb-{{ .Values.name }}-cronjob-maintenance
```

Comment:
```
{{ include "fullName" . }}-maint
```
instead of
```
mariadb-{{ .Values.name }}-cronjob-maintenance
```

Reply:
I'd prefer to keep consistency here. We have the following configmaps and secrets:
`mariadb-{{.Values.name}}-etc` cm
`mariadb-{{.Values.name}}-client` secret
`mariadb-{{.Values.name}}` secret

And temporary ones:
`{{ .Values.name }}-initdb` secret (hook)
`{{ .Values.name }}-configuredb-bin` cm (hook)

Reply:
Ok, then back to `mariadb-{{ .Values.name }}-cronjob-maintenance`